docs/src/content/docs/getting-started.mdx

```diff
@@ -5,33 +5,34 @@ description: Learn how to start building your next documentation site with Starl
 
 import { Tabs, TabItem } from '@astrojs/starlight/components';
 
-## Creating a new project
-
 Starlight is a full-featured documentation theme built on top of the [Astro](https://astro.build) framework.
+This guide will help you get started with a new project.
+See the [manual setup instructions](/manual-setup/) to add the Starlight integration to an existing project.
```

Comment:
Small suggestion to avoid the “theme” => “integration” whiplash observed by @HiDeoo:

```suggestion
See the [manual setup instructions](/manual-setup/) to add Starlight to an existing Astro project.
```